### PR TITLE
SurfaceLights 1.2.3 release

### PR DESCRIPTION
**Pull request by ihsoft**
Note, that I'm taking support of this abandoned mod without explicit consent from the original author. Please, let me know if there is a specific process in such cases.

### CODE REVIEW

**Comment by ihsoft** [comment]
I don't think I get the error. What is wrong?

File: surfacelights/surfacelights-1.2.3.ckan [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    "author": ["Why485", "IgorZ"],
+    "version": "1.2.3",
+    "ksp_version": "1.1.2",
+    "conflicts": [
+        {
+            "name": "SurfaceMountedLights"
+        }
+    ],
+    "download": "http://kerbal.curseforge.com/projects/surface-mounted-stock-alike-lights-for-self/files/2299919/download",
+    "license": "CC-BY-NC-ND-3.0",
```
Comment (plague006) [inline surfacelights/surfacelights-1.2.3.ckan]: No comma on the last line.

**Comment by ihsoft** [comment]
Fixed. Thanks!

**Comment by politas** [comment]
This looks fine as it is, but have you considered pushing full releases to Github? If you included a .version file for specifying KSP compatibility as well, new releases would come to CKAN automatically.

Merging with thanks!

**Comment by politas** [comment]
Oh, and yay! Bob Fitch and I will both be happy to see this available for the latest KSP. More lights!

**Comment by ihsoft** [comment]
Yeah, I was thinking about pushing releases on Github but eventually decided to keep the release files in just one place. It makes things much easier.

**Pull request merged**